src/Core/SettingsChangesHistory.cpp

```diff
@@ -103,6 +104,7 @@ static std::initializer_list<std::pair<ClickHouseVersion, SettingsChangesHistory
             {"enable_variant_type", false, false, "Add alias to allow_experimental_variant_type"},
             {"enable_dynamic_type", false, false, "Add alias to allow_experimental_dynamic_type"},
             {"enable_json_type", false, false, "Add alias to allow_experimental_json_type"},
+            {"distributed_cache_min_bytes_for_seek", false, false, "New private setting."},
```

Comment:
This line seems wrong

Reply:
yep. it was in other version and moved to wrong one